roles/taca/templates/site_fake_delivery.yml.j2

```diff
@@ -0,0 +1,85 @@
+log:
+    file: "{{ ngi_pipeline_site_path }}/log/taca.log"
+
+deliver:
+    rootpath: "/proj/{{ ngi_pipeline_site_delivery }}/nobackup/NGI"
+    analysispath: <ROOTPATH>/ANALYSIS/<PROJECTID>
+    datapath: <ROOTPATH>/DATA/<PROJECTID>
+    stagingpath: <ROOTPATH>/DELIVERY/<PROJECTID>
+    deliverypath: /proj/<UPPNEXID>/INBOX/<PROJECTID>
+    report_aggregate: "ngi_reports ign_aggregate_report -n {{ site_full }} --pandoc_binary"
+    report_sample: "ngi_reports ign_sample_report -n {{ site_full }} --pandoc_binary"
+    reportpath: <ANALYSISPATH>/piper_ngi
+    logpath: <REPORTPATH>/logs
+    deliverystatuspath: <REPORTPATH>/08_misc
+    operator: "{{ recipient_mail }}"
+    hash_algorithm: md5
+    files_to_deliver:
+        -   
+            - <ROOTPATH>/DELIVERY.README.txt
+            - <STAGINGPATH>
+        -
+            - <LOGPATH>/version_report.txt
+            - <STAGINGPATH>/00-Reports
+            - required: True
```

Comment:
In Uppsala, we have some more reports that are delivered, perhaps these could be included with a conditional?

> 
        -
            - <ANALYSISPATH>/qc_sisyphus/reports
            - <STAGINGPATH>/00-Reports/SequenceQC/Sisyphus